packages/workflows/src/handlers/product/remove-products.ts

```diff
@@ -0,0 +1,27 @@
+import { ProductTypes } from "@medusajs/types"
+import { WorkflowArguments } from "../../helper"
+
+export const removeProductsInputAlias = "removeProducts"
+
+export async function removeProducts({
+  container,
+  data,
+}: Omit<WorkflowArguments, "data"> & {
+  data: {
+    removeProductsProducts: ProductTypes.ProductDTO[]
+  }
+}): Promise<void> {
+  const data_ = data.removeProductsProducts
+  if (!data_.length) {
+    return
+  }
+
+  const productModuleService: ProductTypes.IProductModuleService =
+    container.resolve("productModuleService")
+
+  await productModuleService.softDelete(data_.map((p) => p.id))
+}
+
+removeProducts.aliases = {
+  removeProductsProducts: "removeProductsProducts",
```

Comment:
question: I think this is a strange alias name. What's the idea here?

Also, do we need both the alias on line 4 + this one?

Reply:
The idea is to somehow make it unique to prevent any future conflict, but until then I can just have products and we can see later if something needs to be changed

Reply:
I am referring to the double `Products` in the alias. Why not just `removeProducts`? Wouldn't that be unique enough?

Reply:
I will simplify all the namings anyway, but removeProducts was the name of the handler and then a suffix